src/Estranged.Lfs.Adapter.Azure.Blob/AzureBlobAdapterConfig.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace Estranged.Lfs.Adapter.Azure.Blob
+{
+    public class AzureBlobAdapterConfig : IAzureBlobAdapterConfig
+    {
+        public string ConnectionString { get; set; }
+
+        public string ContainerName { get; set; } = "gitlfs";
+
+        public string KeyPrefix { get; set; } = string.Empty;
```

Comment:
Can this be null? The interpolation will just treat it as `string.Empty` so won't have any functional impact, just a better signal that it's not set when looking at the config